.github/scripts/package_push.sh

```diff
@@ -50,8 +52,10 @@ if [[ ${arch} == 'amd64' || ${arch} == 'ppc64el' || ${arch} == 's390x' ]]; then
 
     if [[ "$GITHUB_REF" == 'refs/heads/master' ]]; then 
         docker push ${IMAGE_REPO}/${arch}_anax_rpm:testing
-
         docker tag ${IMAGE_REPO}/${arch}_anax_rpm:testing ${GITHUB_CONTAINER_REGISTRY}/${arch}_anax_rpm:testing
         docker push ${GITHUB_CONTAINER_REGISTRY}/${arch}_anax_rpm:testing
+    else
+        # append the branch name to testing tags for when we're building older versions of anax for testing
+        docker push ${IMAGE_REPO}/${arch}_anax_rpm:testing_${GH_BRANCH}
```

Comment:
The Docker build command produces the image with the `testing` tag. I think you need to tag it with `testing_${GH_BRANCH}` before pushing.